packages/react-interactions/events/src/dom/testing-library/domEventSequences.js

```diff
@@ -156,8 +167,12 @@ export function pointermove(target, payload) {
 export function pointerup(target, defaultPayload = {}) {
   const dispatch = arg => target.dispatchEvent(arg);
   const pointerType = getPointerType(defaultPayload);
-  // eslint-disable-next-line no-unused-vars
-  const {buttons, ...payload} = defaultPayload;
+
+  // Payload could be an array for TouchEvents
+  const payload = defaultPayload;
+  if (defaultPayload.buttons) {
+    delete payload.buttons;
```

Comment:
Delete is bit of an anti-pattern. It's always generally faster to create a mew object wijhout.

Reply:
That's not important in these test utilities, but this code is mutating the original object which it shouldn't